Sources/ArcGISToolkit/Components/Form/FormInputs/SingleLineTextInput.swift

```diff
@@ -62,10 +71,13 @@ struct SingleLineTextInput: View {
         }
         .formTextInputStyle()
         TextInputFooter(
-            currentLength: text.count,
+            text: text,
             isFocused: isFocused,
             element: element,
-            input: input
+            input: input,
+            rangeDomain: rangeDomain,
```

Comment:
This can go away once we have the range domain on the element (`FieldFormElement.domain`).